test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -169,6 +169,7 @@ TEST_P(Http2UpstreamIntegrationTest, BidirectionalStreamingReset) {
   // Send some request data.
   codec_client_->sendData(*request_encoder_, 1024, false);
   ASSERT_TRUE(upstream_request_->waitForData(*dispatcher_, 1024));
```

Comment:
Consider adding this to other tests using different traffic patterns, e.g. test using trailer, etc.

Reply:
Thanks, tests added in protocol_integration_test